ios/Mattermost/main.m

```diff
@@ -2,7 +2,8 @@
 
 #import "AppDelegate.h"
 
-int main(int argc, char * argv[]) {
+int main(int argc, char* argv[])
```

Comment:
Is this change intended?

Reply:
yes, taken from the RN template

Reply:
see the upgrade helper for more details https://react-native-community.github.io/upgrade-helper/?from=0.67.4&to=0.68.0